build_local_windows.ps1

```diff
@@ -53,6 +51,50 @@ python -m venv "$tmpdir/dcos_build_venv"
 
 # Install the DC/OS tools
 ./prep_local_windows.ps1
+# Build a release of DC/OS
+release create $env:USERNAME local_build windows
+
+rm -fo dcos-release.config.yaml
+
+# Creating temp dir, for instance:
+$local_artifacts_dir = "./packages/cache"
+mkdir -f $local_artifacts_dir
+
+##Write DCOS installer locally
+$config_yaml =
+"storage: `
```

Comment:
These escaped newlines are not needed. I'd suggest we remove them now to avoid extra cognitive effort. In the previous stanza defined this way (not changed by this PR) one of the escapes is missing with no bad effect.

Reply:
corrected